### DISCUSSION TITLE
eigener Markercluster - style?

### DISCUSSION THREAD

**Discussion by sfx12** [Tastaturberuf/anyStores#93] [Feature Requests / Funktionswünsche]
ich würde gern die optionen zum markercluster ändern (andere farbe oder eigenes Bild )

Jedoch reagieren Einträge in der Datei "map_google_dynamic.html5" überhaupt nicht !!!
## Ich habe folgenden code benutzt:

var mc = new MarkerClusterer(map, markers, mcOptions);

```
var clusterStyles = [{
        url: 'http://oi59.tinypic.com/snmjj8.jpg',
        height: 150,
        width: 80
    }];

    var mcOptions = {
        gridSize: 20,
        styles: clusterStyles,
        maxZoom: 5
    };
```

---

irgendwie wenn ich auch den kompletten Inhalt der "map_google_dynamic.html5" lösche... bekomme ich im Frontend auch keine Fehler oder jegliches! Wie und wo kann ich testen ob er überhaupt mein Template lädt ?

Vielen Dank schon mal.

lg SfX

**Comment by Tastaturberuf** [comment]
Ich hab keine Ahnung an welcher Stelle man die Clusterbilder tauschen kann.
Das Script ist original aus Github.
